Lnprototest_Testing/README.md

```diff
@@ -27,20 +27,19 @@ To run against Lnprototest BOLT tests:
 
 4. **Set environment variables:**
 
+Do not include the `target/debug/ldk-sample`, this will be automatically added by the runner itself
```

Comment:
```suggestion
   Do not include the `target/debug/ldk-sample`, this will be automatically added by the runner itself
```
Adding space for proper indentation.